tensorflow_io/core/kernels/azfs_kernels.cc

```diff
@@ -154,20 +180,21 @@ std::shared_ptr<azure::storage_lite::storage_credential> get_credential(
   }
 }
 
+// TODO: Enable logging
 azure::storage_lite::blob_client_wrapper CreateAzBlobClientWrapper(
-    const std::string &account) {
+    const std::string& account) {
   azure::storage_lite::logger::set_logger(
-      [](azure::storage_lite::log_level level, const std::string &log_msg) {
+      [](azure::storage_lite::log_level level, const std::string& log_msg) {
         switch (level) {
           case azure::storage_lite::log_level::info:
-            _TF_LOG_INFO << log_msg;
+            // _TF_LOG_INFO << log_msg;
```

Comment:
Is there any reason why these logs are commented out?

Reply:
@terrytangyuan The logging will require the plugin.so depends on C++ APIs in tensorflow which will makes cross-version (e.g., 0.16.0 support TF 2.4, 2.5, 2.6). So this line is temporarily disabled. Once TF exposes C API for logging we will add it back (using C API instead).

Reply:
Sounds good.